bears/planemo/PlanemoLintBear.py

```diff
@@ -0,0 +1,34 @@
+import os
+import shlex
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='planemo',
+        output_format='regex',
+        output_regex=r'..\s(?P<severity>\w*:)(?P<message>.*)',
+        severity_map={'WARNING:': RESULT_SEVERITY.MAJOR,
+                      'CHECK:': RESULT_SEVERITY.NORMAL,
+                      'INFO:': RESULT_SEVERITY.INFO})
+class PlanemoLintBear:
+    """
+    Checks the code with planemo lint. This will run
```

Comment:
The description needs to improve IMO. A naiive user wouldn't understand much by the given description.